cmd/oceantv/broadcast/storage.go

```diff
@@ -130,9 +130,16 @@ func objTok(ctx context.Context, url string) (*oauth2.Token, error) {
 // fileTok extracts an oauth2.Token from a file with name specified by the given
 // URL.
 func fileTok(uri string) (*oauth2.Token, error) {
-	_, name, err := googleStorageAddr(uri)
-	if err != nil {
-		return nil, fmt.Errorf("could not parse uri: %w", err)
+	c := os.Getenv("YOUTUBE_API_CREDENTIALS")
+	var name string
```

Comment:
You could simplify by defaulting name, then overriding when it is empty, i.e,

```
	name := os.Getenv("YOUTUBE_API_CREDENTIALS")
	var err error
	if name == "" {
		_, name, err = googleStorageAddr(uri)
		if err != nil {
			return nil, fmt.Errorf("could not parse uri: %w", err)
		}
	}

```

Reply:
Done.